packages/nakama-js/socket.ts

```diff
@@ -765,7 +767,7 @@ export class DefaultSocket implements Socket {
 
     this.adapter.onMessage = (message: any) => {
       if (this.verbose && window && window.console) {
-        console.log("Response: %o", JSON.stringify(message));
+        console.log("Response: %o", atob(JSON.stringify(message)));
```

Comment:
This will base64 decode messages that have not been base64 encoded

Reply:
If I'm reading `web_socket_adapter.ts` right, the encoding happens in there (lines 162 - 166). Would it be better to remove the encoding from there and do it in the send method of `socket.ts`?

Reply:
I'm not sure if that's related to the issue here -- we want to log the base64 decoded version of the match data payload, right? So what if you just base64 decode the match_data or party_data within verbose mode, and log an object with that decoded data?

Reply:
@lugehorsam take a look, should be better now